packages/ontario-frontend-cli/commands/ontario-create-app/handleCreateApp.js

```diff
@@ -25,49 +25,66 @@ const { createOntarioAppQuestions } = require('../../core/questions');
 configureTemplates(CREATE_TEMPLATE_DIR);
 
 async function createNewProject(answers, options) {
+  logger.debug('Answers received:', answers);
+  logger.debug('Options received:', options);
+
   const newProjectPath = path.resolve(process.cwd(), answers.projectName);
 
   // Create the directory for the new project
   logger.info(`Creating a new Ontario Frontend project in ${newProjectPath}`);
+  logger.debug(`Project path resolved to: ${newProjectPath}`);
+
   ensureDirectory(newProjectPath);
+  logger.debug(`Directory created: ${newProjectPath}`);
 
-  const ontarioFrontendDependency = options.isLocal ? `"file:${LOCAL_CORE_DEPENDENCY_DIR}/"` : '"latest"';
+  const ontarioFrontendDependency = options.isLocal
+    ? `"file:${LOCAL_CORE_DEPENDENCY_DIR}/"`
+    : '"latest"';
 
   // Configuration for the new project
   const conf = {
     ...answers, //Directly spread relevant answers
     createDate: new Date().toISOString(),
-    ontarioFrontendDependency: ontarioFrontendDependency
+    ontarioFrontendDependency: ontarioFrontendDependency,
   };
 
+  logger.debug('Project configuration:', conf);
+
   logger.success('Created project context');
 
   // Copy boilerplate files
   await copyBoilerplateFiles(newProjectPath);
+  logger.debug('Boilerplate files copied successfully');
 
   // Generate project files
   await generateProjectFiles(newProjectPath, conf);
+  logger.debug('Project files generated successfully');
 
   // Copy ESLint config if opted-in
   if (conf.addESLint) {
     try {
       await handlePackageCopy(newProjectPath, 'eslint');
+
+      logger.debug('ESLint configuration copied successfully');
     } catch (err) {
-      throw err
+      throw new Error(`Failed to copy ESLint configuration: ${err.message}`);
     }
   }
 
   // Copy Prettier config if opted-in
   if (conf.addPrettier) {
     try {
       await handlePackageCopy(newProjectPath, 'prettier');
+
+      logger.debug('Prettier configuration copied successfully');
     } catch (err) {
-      throw err
+      throw new Error(`Failed to copy Prettier configuration: ${err.message}`);
     }
   }
 
   // Install npm dependencies, including the core Frontend dependency
   await installAllPackages(newProjectPath);
+  logger.debug('npm dependencies installed successfully');
```

Comment:
Very minor but we seem to capitalize NPM in the success message elsewhere, the debug message should probably match the capitalized sentence case.
<img width="452" alt="Screenshot 2024-06-06 at 2 14 05 PM" src="https://github.com/ongov/ontario-frontend/assets/164547352/c326d269-014a-49e6-99ef-7687329dad6d">

Reply:
Removing the message from here as it provides no value.